unified_planning/test/pddl/operator/problem.pddl

```diff
@@ -0,0 +1,22 @@
+(define (problem operator_test_problem) (:domain operator_test_domain)
+(:objects
+    a b c
+)
+(:init
+  (connected a b)
+  (connected b c)
+  (at a)
+)
+
+(:goal
+  (and
+    (forall 
+      (?x)
+      (and
+        (at ?x)
+        (forall (?y) (connected ?x ?y))
+      )
+    )
+  )
```

Comment:
It seems an impossible problem, can we update it with a problem that makes sense and has nested quantifiers?